doc/download_example_data.py

```diff
@@ -0,0 +1,9 @@
+from download import download
+import os
+
+url = "http://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/nybb_16a.zip"
+
+path_data = os.path.join('..', 'gallery', 'data')
+path_zip = os.path.join(path_data, 'nybb_16a.zip')
+download(url, path_zip)
+download(url, path_data, zipfile=True)
```

Comment:
This is not needed anymore. In the meantime, the nybb dataset is available from `geopandas.datasets.get_path('nybb')`